openlattice.yaml

```diff
@@ -2556,6 +2556,136 @@ paths:
         "200":
           description: Success
 
+  ##################
+  ## LINKING APIs ##
+  ##################
+  
+  /datastore/linker/linkingfeedback:
```

Comment:
this path seems wrong... this api isn't in datastore

Reply:
I moved it to linker from datastore a while back. Path should be `linker/linkingfeedback`.
Same goes for all the paths below

Reply:
done !